package.json

```diff
@@ -73,6 +78,42 @@
 			{
 				"command": "extension.configureLaunchDarkly",
 				"title": "LaunchDarkly: Configure"
+			},
+			{
+				"command": "ldFeatureFlags.refreshEntry",
+				"title": "LaunchDarkly: Refresh Feature Flags",
+				"icon": {
+					"light": "resources/light/refresh.svg",
+					"dark": "resources/dark/refresh.svg"
+				}
+			},
+			{
+				"command": "ldFeatureFlags.copyKey",
+				"title": "LaunchDarkly: Copy Key to Clipboard",
+				"icon": {
+					"light": "resources/light/files.svg",
+					"dark": "resources/dark/files.svg"
+				}
+			},
+			{
+				"command": "launchdarkly.treeviewrefresh",
```

Comment:
What's the difference between `refreshEntry` and `treeviewrefresh`? Since we're connecting to the flags stream, I'd recommend removing any manual refresh options and favoring automatically refreshing whenever flags are updated